2022-03-03 13:47:27,684 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2022-03-03 13:47:27,685 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from C:\Python39\lib\site-packages\instabot\bot\bot.py
2022-03-03 13:47:27,686 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2022-03-03 13:47:28,045 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2022-03-03 13:47:28,045 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2022-03-03 13:47:28,047 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2022-03-03 13:47:28,048 - instabot version: 0.117.0 (api) - WARNING - That means 'too many requests'. I'll go to sleep for 5 minutes.
2022-03-03 13:47:28,805 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2022-03-03 13:47:28,808 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from C:\Python39\lib\site-packages\instabot\bot\bot.py
2022-03-03 13:47:28,811 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2022-03-03 13:47:28,984 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2022-03-03 13:47:28,985 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2022-03-03 13:47:28,986 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2022-03-03 13:47:28,988 - instabot version: 0.117.0 (api) - WARNING - That means 'too many requests'. I'll go to sleep for 5 minutes.
